addons/monit/00_packetfence.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+ALERT_EMAIL=$1
+SUBJECT_NAME=$2
+
+if [ -z "$ALERT_EMAIL" ] || [ -z "$SUBJECT_NAME" ]; then
+    echo 'Missing parameter(s)\n'
+    echo 'Syntax should be: ./00_packetfence.sh "alerting@email.address" "subject identifier"'
+    exit 1;
+fi
+
+cat > /etc/monit.d/packetfence.monit << EOF
+set mailserver localhost
+set alert $ALERT_EMAIL
+
+set mail-format {
+    from: monit@\$HOST
+    subject: $SUBJECT_NAME | Monit Alert -- \$EVENT on resource '\$SERVICE'
+    message:
+    Date:        \$DATE
+    Host:        \$HOST
+    Event:       \$EVENT
+    Resource:    \$SERVICE
+    Action:      \$ACTION
+    Description: \$DESCRIPTION
+
+    A copy of this alert have been sent to the PacketFence support team.
```

Comment:
I don't want to receive all the monit alert from all the packetfence installation